rosetta/Dockerfile.gemma

```diff
@@ -0,0 +1,71 @@
+# syntax=docker/dockerfile:1-labs
+
+ARG BASE_IMAGE=ghcr.io/nvidia/jax-mealkit:jax
+ARG URLREF_GEMMA=https://github.com/google-deepmind/gemma.git#main
+ARG SRC_PATH_GEMMA=/opt/gemma
+ARG URLREF_BIG_VISION=https://github.com/google-research/big_vision.git#main
+ARG SRC_PATH_BIG_VISION=/opt/big_vision
+ARG URLREF_COMMON_LOOP_UTILS=https://github.com/google/CommonLoopUtils.git#main
+ARG SRC_PATH_COMMON_LOOP_UTILS=/opt/common_loop_utils
+ARG URLREF_FLAXFORMER=https://github.com/google/flaxformer.git#main
+ARG SRC_PATH_FLAXFORMER=/opt/flaxformer
+ARG URLREF_PANOPTICAPI=https://github.com/akolesnikoff/panopticapi.git#mute
+ARG SRC_PATH_PANOPTICAPI=/opt/panopticapi
+
+###############################################################################
+## Download source and add auxiliary scripts
+###############################################################################
+
+FROM ${BASE_IMAGE} as mealkit
+ARG URLREF_GEMMA
+ARG SRC_PATH_GEMMA
+ARG URLREF_BIG_VISION
+ARG SRC_PATH_BIG_VISION
+ARG URLREF_COMMON_LOOP_UTILS
+ARG SRC_PATH_COMMON_LOOP_UTILS
+ARG URLREF_FLAXFORMER
+ARG SRC_PATH_FLAXFORMER
+ARG URLREF_PANOPTICAPI
+ARG SRC_PATH_PANOPTICAPI
+
+
+RUN <<"EOF" bash -ex
+git-clone.sh ${URLREF_GEMMA} ${SRC_PATH_GEMMA}
+git-clone.sh ${URLREF_BIG_VISION} ${SRC_PATH_BIG_VISION}
+git-clone.sh ${URLREF_COMMON_LOOP_UTILS} ${SRC_PATH_COMMON_LOOP_UTILS}
+git-clone.sh ${URLREF_FLAXFORMER} ${SRC_PATH_FLAXFORMER}
+git-clone.sh ${URLREF_PANOPTICAPI} ${SRC_PATH_PANOPTICAPI}
+echo "-e file://${SRC_PATH_GEMMA}" >> /opt/pip-tools.d/requirements-gemma.in
+echo "-e file://${SRC_PATH_COMMON_LOOP_UTILS}" >> /opt/pip-tools.d/requirements-gemma.in
+echo "-e file://${SRC_PATH_FLAXFORMER}" >> /opt/pip-tools.d/requirements-gemma.in
+echo "-e file://${SRC_PATH_PANOPTICAPI}" >> /opt/pip-tools.d/requirements-gemma.in
+echo "jupyterlab
+gcloud
+overrides
+ml_collections
+einops~=0.7
+sentencepiece
+kagglehub
+absl-py
+distrax
+optax
+protobuf
+tfds-nightly
+tensorflow
+tensorflow-text
+tensorflow-gan
+" >> /opt/pip-tools.d/requirements-gemma.in
+EOF
+
+ENV PYTHONPATH "/opt/big_vision/:${PYTHONPATH}"
```

Comment:
It's a bit unfortunate that they didn't make this big_vision project installable.
First, this should be `${SRC_PATH_BIG_VISION}` rather than hardcoded.
Second, should we be linking https://github.com/google-research/big_vision/blob/70b22bb60c46023e50fd0f5d42e0f48f8dfe2fdb/big_vision/requirements.txt into `/opt/pip-tools.d`? Maybe then some of the above hardcoded requirements can be dropped?

Reply:
1. Done
2. Unfortunately, it does not work, b/c Big Vision requirements contains packages we cannot process in our pip-compile approach (for example, `git+https://github.com/google/flaxformer`).

Reply:
What's wrong with the second one? We already have explicit handling for pip vcs installations like that?